tests/sensors/test_external_task_sensor.py

```diff
@@ -1049,14 +1050,19 @@ def dag_bag_head_tail():
     dag_bag = DagBag(dag_folder=DEV_NULL, include_examples=False)
 
     with DAG("head_tail", start_date=DEFAULT_DATE, schedule="@daily") as dag:
+        @task
+        def dummy_task(x: int):
+            return x
```

Comment:
How about you don't modify this fixture but create a separate fixture for this PR? If I understand correctly there's no breaking change so this fixture should still work

Reply:
sure. in fact i'll remove this change entirely since clearing parent dags will not clear the mapped tasks in child dags right now. see  https://github.com/apache/airflow/issues/27462